### ISSUE TITLE
group-by-v2 fold not working(摺疊沒有效果)

### ISSUE THREAD

**Issue by s961744** [wenzhixin/bootstrap-table#3897]
[jsfiddle](https://jsfiddle.net/s961744/3gdj74e1/9/)

Reference to [example](http://issues.wenzhixin.net.cn/bootstrap-table/#extensions/group-by-v2.html)
It did group the data rows, but didn't fold when clicking the group rows.
Please help me with this issue, thank you.

我參考[範例](http://issues.wenzhixin.net.cn/bootstrap-table/#extensions/group-by-v2.html)
資料確實有分群組，但是按下群組的資料時沒有摺疊起來
請幫幫我，謝謝

**Comment by LianJilu** [comment]
you can view the source, and check your dependency is right
![image](https://user-images.githubusercontent.com/25577583/43075668-0114f4fa-8eb4-11e8-936f-f68a39efc303.png)

**Comment by s961744** [comment]
Thank  you, @LianJilu 

After a lot of tests, I finally found out that it was caused by the version of Bootstrap.css.
It worked well on all the version of Bootrap 3 , even the last version v3.37,
but not working on v4.00...
Updated  [jsfiddle](https://jsfiddle.net/s961744/3gdj74e1/19/)
The only change is the link of bootstrap.mis.css changed to https://stackpath.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css

Does this mean I could only use Bootstrap 3 on this page?
Is there anyone working group-by-v2 well on Bootstrap 4?
